src/main/java/net/minecraftforge/fml/common/ObfuscationReflectionHelper.java

```diff
@@ -60,6 +67,22 @@
         return mappedNames;
     }
 
+    public static String remapFieldName(Class<?> clazz, String fieldName)
```

Comment:
Maybe make it more clear that this us supposed to be srg name by naming it "fieldSrgName"? (and same for methods)